modules/tobira/src/main/java/org/opencastproject/tobira/impl/Item.java

```diff
@@ -125,6 +125,12 @@ class Item {
 
       final var captions = findCaptions(mp);
 
+      // Get the generated slide text.
+      final var slideText = Arrays.stream(mp.getElements())
+          .filter(mpe -> mpe.getFlavor().eq("mpeg-7/text"))
+          .map(element -> element.getURI())
+          .findFirst();
```

Comment:
I wonder... what if there are multiple of these elements? You just take the first here. Can anyone say whether in the real world, there might be multiple such elements?

But this isn't a blocker. I also just trust Matthias with the `mpeg-7/text` filter. I don't have enough experience to say whether that filters for less or more than what we want. :shrug: